src/API/WorkController.ts

```diff
@@ -4,8 +4,8 @@ import { Collection, Db } from 'mongodb'
 import * as Pino from 'pino'
 
 import { childWithFileName } from 'Helpers/Logging'
-import { Exchange } from 'Messaging/Messages'
 import { Messaging } from 'Messaging/Messaging'
+import { ExchangeConfiguration } from './ExchangeConfiguration'
```

Comment:
Can you add an empty line between absolute-path imports (`import { Messaging } from 'Messaging/Messaging'`) and relative-path imports?